ci-operator/config/openshift/ocs-operator/openshift-ocs-operator-master__latest-upstream.yaml

```diff
@@ -0,0 +1,135 @@
+      - destination_dir: .
+        source_path: /go/src/github.com/openshift/ocs-operator/metrics/deploy/prometheus-ocs-rules.yaml
+  to: ocs-operator
+- dockerfile_path: openshift-ci/Dockerfile.registry.build
+  from: operator-registry
+  to: ocs-registry
+- dockerfile_path: build/Dockerfile
+  from: os
+  inputs:
+    bin:
+      paths:
+      - destination_dir: _output/bin
+        source_path: /go/src/github.com/openshift/ocs-operator/build/_output/bin/ocs-operator
+      - destination_dir: _output/bin
+        source_path: /go/src/github.com/openshift/ocs-operator/build/_output/bin/metrics-exporter
+      - destination_dir: _output
+        source_path: /go/src/github.com/openshift/ocs-operator/metrics/deploy/prometheus-ocs-rules-external.yaml
+      - destination_dir: .
+        source_path: /go/src/github.com/openshift/ocs-operator/metrics/deploy/prometheus-ocs-rules.yaml
+  to: ocs-operator-quay
+- dockerfile_path: openshift-ci/Dockerfile.registry
+  from: operator-registry
+  to: ocs-registry-quay
+- dockerfile_path: must-gather/Dockerfile
+  from: must-gather
+  inputs:
+    src:
+      paths:
+      - destination_dir: must-gather
+        source_path: /go/src/github.com/openshift/ocs-operator/must-gather/Dockerfile
+      - destination_dir: .
+        source_path: /go/src/github.com/openshift/ocs-operator/must-gather/collection-scripts
+      - destination_dir: .
+        source_path: /go/src/github.com/openshift/ocs-operator/must-gather/templates
+  to: ocs-must-gather-quay
+operator:
+  bundles:
+  - as: dev-master-dependencies
+    dockerfile_path: openshift-ci/Dockerfile.bundle.master
+  substitutions:
+  - pullspec: quay.io/ocs-dev/ocs-operator:latest
+    with: ocs-operator
+  - pullspec: ceph/daemon-base:.*
+    with: ceph-master
+  - pullspec: rook/ceph:.*
+    with: rook-master
+promotion:
+  excluded_images:
+  - ocs-operator
+  - operator-registry-base
+  - ocs-registry
+  - ocs-registry-master
+  name: ocs-4.9
+  namespace: ocp
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tag_specification:
+  name: "4.8"
+  namespace: ocp
+tests:
+- as: ocs-operator-bundle-latest-master-e2e-aws
+  cron: 0 0 * * *
+  steps:
+    cluster_profile: aws
+    dependencies:
+      OO_INDEX: ci-index-dev-master-dependencies
+    env:
+      OO_CHANNEL: alpha
+      OO_INSTALL_NAMESPACE: openshift-storage
+      OO_PACKAGE: ocs-operator
+      OO_TARGET_NAMESPACES: '!install'
+    post:
+    - as: ocs-must-gather
+      cli: latest
+      commands: hack/dump-debug-info.sh
+      from: src
+      resources:
+        requests:
+          cpu: 100m
+    test:
+    - as: tests
+      commands: |
+        export GOFLAGS=''; make functest
+      dependencies:
+      - env: OO_INDEX
+        name: ci-index
+      from: src
+      resources:
+        requests:
+          cpu: 100m
+          memory: 200Mi
+    workflow: optional-operators-ci-aws
```

Comment:
@BlaineEXE This is probably causing Post Install steps to be run before actual Tests. Can you verify?

Reply:
I'm not sure how to verify. Could you elaborate? This is just copied from Elise's prior work, not net new code.

Reply:
In the docs, I don't see any alternative to the `optional-...` workflows. https://docs.ci.openshift.org/docs/how-tos/testing-operator-sdk-operators/

Reply:
Also, I don't think `optional-` means the test step is optional. It is that it is an "optional operator"

https://steps.ci.openshift.org/workflow/optional-operators-ci-aws